lib/generators/statesman/generator_helpers.rb

```diff
@@ -46,7 +46,7 @@ def database_supports_partial_indexes?
     end
 
     def metadata_default_value
```

Comment:
We might as well drop this method and put the correct value directly in the template if we're convinced it works everywhere.